packages/react/src/components/utils/index.tsx

```diff
@@ -27,6 +27,21 @@ export const createForwardRef = <PropType, ElementType>(
   return React.forwardRef(forwardRef);
 };
 
+export const mergeRefs = (
+  ...refs: (React.ForwardedRef<any> | React.Ref<any> | undefined)[]
+) => {
+  return (value: any) => {
+    refs.forEach(ref => {
```

Comment:
Do we need to return anything here?

Reply:
I don't believe react has any requirements or expectations for the return value of a callback ref, but I will double-check and confirm.

Reply:
Confirmed the docs referenced above show an example with no return value, and the typescript types show the ForwardedRef function format and RefCallback should return void.